elm-analyse.json

```diff
@@ -0,0 +1,6 @@
+{
+    "checks": {
+        "UnusedPatternVariable": false
+    },
+    "exclude": [ "src/Library/Tailwind.elm" ]
```

Comment:
This makes the ElmLS more responsive for me. Figured, I could leave this in here for others to enjoy until ElmLS performance issues are resolved.

Reply:
Yeah that's fine, shouldn't be analysed anyway.